pandas/tests/plotting/test_series.py

```diff
@@ -686,7 +686,7 @@ def test_time_series_plot_color_with_empty_kwargs(self):
         ncolors = 3
 
         _, ax = self.plt.subplots()
-        for i in range(ncolors):
+        for _ in range(ncolors):
```

Comment:
maybe intended to pss the arg to s.plot?

Reply:
The subplot seems to have only one axis. I think this test makes sure that matplotlib cycles through its colors when called multiple times.